packages/design-system/src/components/text-field.tsx

```diff
@@ -55,6 +55,22 @@ export const TextFieldIconButton = styled(
             color: "white",
           },
         },
+        set: {
```

Comment:
like I said in a PR before, text field has no idea of set/inherited semantics, its a component that is used outside of style panel.

I don't know why its not clear automatically.

Reply:
create your own component in css value input:

```ts
const CssValueInputIconButton = styled(TextFieldIconButton, {
  variants: {
    set: {},
    inherited: {}
  }
})
```

Reply:
or something like this

Reply:
active is a generic state for this icon button, its the same across all usages, not just style panel